netlib/tcp.py

```diff
@@ -708,6 +715,7 @@ def connect(self):
         self.connection = connection
         self.ip_address = Address(connection.getpeername())
         self._makefile()
+        return _closer(self)
```

Comment:
Well, there's a reason why we don't just connect() int he __init__ of TCPClient. It's just a more flexible API. This is especially the case with derived classes like Pathoc where reuse is quite often useful.